server/images.go

```diff
@@ -532,6 +533,10 @@ func CreateModel(ctx context.Context, name model.Name, modelFileDir, quantizatio
 				}
 			}
 
+			return true
+		case "application/vnd.ollama.image.template":
```

Comment:
This is ugly but I want to rewrite this part at some point with rangefuncs

Reply:
Is it guaranteed that the last template is the one specified by the user?

Reply:
No it's an assumption. I refactored it so it no longer makes that assumption and the template is only detected when importing from a file or directory